roles/dns/manage-dns-records/tasks/route53/process-records.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: "Print WARNING for views other than 'public' and 'private'"
```

Comment:
This still prints even if we are only provisioning a named server.

Reply:
yeah, I'm starting to question the "combined" role for multiple DNS servers as it's presenting various challenges. I think for now, let's move forward with it as-is, and we can do a follow-up PR to split into multiple roles within the "dns" directory.

Reply:
ok, sounds good